source/locale/pt_BR/LC_MESSAGES/components-and-apis.po

```diff
@@ -409,8 +409,9 @@ msgstr ""
 "APIs em :doc:`./using-api-interface`."
 
 #: ../../source/components-and-apis.rst:133
-msgid "*Exposed* endpoints"
-msgstr "*Endpoints* expostos"
+#, fuzzy
```

Comment:
When "#, fuzzy" appears, it is because there is something pending in the translation, the correct thing is to remove all of them.